plugins/gestures.koplugin/main.lua

```diff
@@ -466,24 +468,55 @@ Default value: %1]]), Screen.low_pan_rate and 5.0 or 30.0),
                 end,
                 separator = true,
             },
+            {
+                text = _("Tap interval"),
+                keep_menu_open = true,
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local GestureDetector = require("device/gesturedetector")
+                    local items = SpinWidget:new{
+                        title_text = _("Tap interval"),
+                        info_text = T(_([[
+Another tap made in this interval after a first tap will be considered a bounce and ignored.
```

Comment:
Maybe something like this?
```suggestion
Any other taps made within this interval after a first tap will be considered accidental and ignored.
```